lib/bridge/MatrixHandler.js

```diff
@@ -674,6 +679,14 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
             return;
         }
     }
+    else if (cmd === "!bridgeversion") {
+        const msgText = `BridgeVersion: ` + BridgeVersion ;
```

Comment:
I'm a bit impressed the linter doesn't complain about the space before the semicolon...

As for the line: it feels a bit weird to use a templated string to append a variable.